docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,80 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* Improve client protocol throughput by introducing the spooling protocol.
+  ({issue}`24214`)
+* Improve performance of querying `system.jdbc.tables` for multiple catalogs.
+  ({issue}`24159`)
+* Add support for connectors to improve performance for `MERGE` statements by
+  only changing updated columns. ({issue}`24075`)
+* Add support for changing field types inside a map. ({issue}`24248`)
+* Replace Python-based launcher script with fully-compatible, go-based
+  executable. Removes the requirement for a Python installation on Trino cluster
+  nodes. ({issue}`24271`)
+* Improve error messages for `INTERVAL` literals. ({issue}`23002`)
```

Comment:
Remove this entry. We don't document improvements to error messages, as they are self-evident.